hack/lib/common.sh

```diff
@@ -539,7 +544,18 @@ function kube::common::start_controller_manager {
           --cert-dir="${CERT_DIR}" \
           --default-network-template-path="${ARKTOS_NETWORK_TEMPLATE}" >"${CTLRMGR_LOG}" 2>&1 &
       else
-        KUBE_CONTROLLERS="*,-nodelifecycle,-nodeipam"
+        KUBE_CONTROLLERS="*,-daemonset,-nodelifecycle,-nodeipam,-ttl"
```

Comment:
are we going to start daemonset controller now?

Reply:
> are we going to start daemonset controller now?

Yes, we can start the controller. But we will disable daemonset test in 430.